src/Microsoft.IdentityModel.Protocols.OpenIdConnect/Configuration/OpenIdConnectConfiguration.cs

```diff
@@ -360,5 +360,228 @@ public OpenIdConnectConfiguration(string json)
         /// </summary>
         [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore, NullValueHandling = NullValueHandling.Ignore, PropertyName = OpenIdProviderMetadataNames.UserInfoSigningAlgValuesSupported, Required = Required.Default)]
         public ICollection<string> UserInfoEndpointSigningAlgValuesSupported { get; } = new Collection<string>();
+
+#region shouldserialize
+        /// <summary>
+        /// Gets a bool that determines if the 'acr_values_supported' (AcrValuesSupported) property should be serialized.
+        /// This is used by Json.NET in order to conditionally serialize properties.
+        /// </summary>
+        /// <return>true if 'acr_values_supported' (AcrValuesSupported) is not empty; otherwise, false.</return>
+        public bool ShouldSerializeAcrValuesSupported()
```

Comment:
@mafurman can you add these declarations: [EditorBrowsable(EditorBrowsableState.Never)]

Reply:
Just added them in!